lib/ferto/client.rb

```diff
@@ -73,6 +73,7 @@ def initialize(opts = {})
     #     aggr_proxy: 'http://myproxy.com/',
     #     user_agent: 'my-useragent',
     #     mime_type: "image/jpeg",
+    #     request_headers: "image/*,*/*;q=0.8",
```

Comment:
The yard type specifier above also needs update.

Reply:
Is that right? Shouldn't request_headers be a hash?

Reply:
Ooops! Yes you are right! Sorry, will fix it!